AutomatedTesting/Gem/PythonTests/smoke/test_CLITool_PythonBindingsExample_Works.py

```diff
@@ -14,7 +14,7 @@
 import subprocess
 
 
-@pytest.mark.SUITE_smoke
+@pytest.mark.skip(reason="GHI #13693: Test Periodically Fails")
```

Comment:
Is there a reason you're not sandboxing this test instead? It can be useful to have the test failure info from sandboxed tests when going back to fix them.

Reply:
I can do that if it's preferred, could you please remind me of the syntax for that? Thanks!

Reply:
mark SUITE_sandbox and then make sure the cmakelists.txt file that includes this PY file handles that correctly. I don't think there's a need to go back and change this, the person who plans to fix it can do that if they need to next.